test/support/video_compositor_helpers.ex

```diff
@@ -0,0 +1,43 @@
+defmodule VideoCompositor.Helpers do
```

Comment:
Rename the module, maybe to something like `Membrane.VideoCompositor.FFmpeg.Utils` (or `-||-.Helpers).

Reply:
This functionality can be usefull for other implementations' tests. However, it can be better named.